apps/web/src/state/pools/index.ts

```diff
@@ -210,9 +211,19 @@ export const fetchPoolsPublicDataAsync = (chainId: number) => async (dispatch, g
       const stakingTokenAddress = isAddress(pool.stakingToken.address)
       let stakingTokenPrice = stakingTokenAddress ? prices[stakingTokenAddress] : 0
       if (stakingTokenAddress && !prices[stakingTokenAddress] && !isPoolFinished) {
-        // eslint-disable-next-line no-await-in-loop
-        const result = await fetchTokenUSDValue(chainId, [stakingTokenAddress])
-        stakingTokenPrice = result.get(stakingTokenAddress) || 0
+        // TODO: Remove this when fetchTokenUSDValue can get APL USD Price
+        if (
+          pool.stakingToken.chainId === ChainId.ARBITRUM_ONE &&
+          pool.stakingToken.address === arbitrumTokens.alp.address
+        ) {
+          // eslint-disable-next-line no-await-in-loop
+          const result = await fetchTokenAplPrice()
+          stakingTokenPrice = result
```

Comment:
no need for an extra `result` variable?

Reply:
yeah we can simply write `stakingTokenPrice = await fetchTokenAplPrice()`

Reply:
done